myapp/static/css/main.css

```diff
@@ -37,4 +37,9 @@ body {
 
 .clickable-row {
     cursor: pointer;
+}
+.row-m-t{
+   margin-top: 30px;
+  margin-bottom: 30px;
```

Comment:
In my opinion, the top and bottom margins are a bit too large. Somewhere around 16px would be better.

Reply:
updated